macros/PGauxiliaryFunctions.pl

```diff
@@ -107,72 +107,38 @@ sub Round {
 
 #least common multiple
 #VS 6/29/2000
+# made recursive to handle the gcf of any number of inputs 5/16/2020
```

Comment:
There is no need to add comments explaining changes and dates.  The git repository shows the changes.  The dates that are here are from an old pre-git practice that should be discontinued.

Reply:
Thanks for reviewing this. OK, but should I delete the note that are already there? Whoever VS is, is probably no longer the person to consult with a question. (Assuming that is the reason for this kind of commentary historically.)